include/deal.II/base/symmetric_tensor.h

```diff
@@ -431,7 +441,7 @@ namespace internal
 
 
 /**
- * Provide a class that stores symmetric tensors of rank 2,4,... efficiently,
+ * Provide a class that stores symmetric tensors of rank 2,4,\ldots efficiently,
```

Comment:
If you want to use `\ldots` here, then you need to enclose things into `$...$`. I guess using `...` is fine here, since it's not a typeset formula.